### PR TITLE
npmd: fix npmd config parsing and server command strings

### PR DESCRIPTION
**Pull request by chparimi**
1. NPMD UI sends agent subnet value as an int as opposed to string
2. Alter command strings to match those defined in DSC module

### CODE REVIEW

**Comment by chparimi** [comment]
@Microsoft/omsagent-devs

**Comment by varastog** [comment]
pbuild succeeded

File: test/code/plugins/in_npmd_server_plugintest.rb [hunk 1]
```diff
@@ -28,10 +28,10 @@ class NPMDServerTest < Test::Unit::TestCase
 
     CMD_ENUMERATE_BINARY_INSTANCES = "ps aux | grep fake_npmd"
 
-    CMD_START_NPMD  = "StartNPMD"
-    CMD_STOP_NPMD   = "StopNPMD"
-    CMD_CONFIG_NPMD = "ConfigNPMD"
-    CMD_PURGE_NPMD  = "PurgeNPMD"
+    CMD_START_NPMD  = "StartNPM"
```
Comment (robbiezhang) [inline test/code/plugins/in_npmd_server_plugintest.rb]: if they should be the same as the one defined in the in_npmd_server.rb, please keep 1 copy.

Comment (chparimi) [inline test/code/plugins/in_npmd_server_plugintest.rb]: Done.

**Pull request merged**